scipy/signal/signaltools.py

```diff
@@ -408,24 +401,13 @@ def fftconvolve(in1, in2, mode="full", axes=None):
     fslice = tuple([slice(sz) for sz in shape])
     # Pre-1.9 NumPy FFT routines are not threadsafe.  For older NumPys, make
     # sure we only call rfftn/irfftn from one thread at a time.
```

Comment:
This comment is stale.

Reply:
This is fixed in #10361, I'm trying not to have too much overlap in changes

Reply:
Actually maybe it's not... I'll remove it in one of these PRs at least.